box/docker-compose.supabase.yml

```diff
@@ -0,0 +1,530 @@
+# Usage
```

Comment:
Just wondering how you produced this file, was it mainly an example from supabase docs or something?